### PR TITLE
Automatically check if system needs DCH drivers

### LINKED ISSUE #63: Implement support for DCH drivers
DCH Driver compability.
(DCH Drivers are common on Laptops that came with Win10 Preinstalled)

""DCH" (Declarative, Componentized, Hardware Support Apps) refers to new packages preinstalled by OEMS implementing the Microsoft Universal Driver paradigm.

DCH drivers cannot be installed over a standard system, and Standard drivers cannot be installed over a DCH system."

(See:
https://docs.microsoft.com/en-us/windows-hardware/drivers/what-s-new-in-driver-development#windows-driver-frameworks-wdf
)

### PR DESCRIPTION
**Pull request by Osspial**
Based on solution found here:
https://www.reddit.com/r/PowerShell/comments/cbjoz9/powershell_to_find_nvidia_dch_or_standard_driver/. Fixes #63.

I don't do very much C# programming, so please let me know if there are style issues with this PR you'd like me to address. Also, would it be possible to do a `1.14.0` release once this has been merged?

### CODE REVIEW

File: TinyNvidiaUpdateChecker/MainConsole.cs [hunk 1]
```diff
@@ -554,6 +557,33 @@ private static void GpuInfo()
                 Console.WriteLine(ex.ToString());
             }
 
+            // determine whether or not we need a DCH driver
+            try
+            {
```
Comment (ElPumpo) [inline TinyNvidiaUpdateChecker/MainConsole.cs]: If you look at the existing code, it doesn't have newlines for the `{ }` characters

For example, you wrote like this
```csharp
}
else
{
```
I write like this

```csharp
} else {
```

File: TinyNvidiaUpdateChecker/MainConsole.cs [hunk 2]
```diff
@@ -97,6 +98,8 @@ class MainConsole
         /// </summary>
         private static int osID;
 
+        private static int dtcID;
```
Comment (ElPumpo) [inline TinyNvidiaUpdateChecker/MainConsole.cs]: No `<summary>` :/

**Comment by Osspial** [comment]
Review comments should be addressed. Thank you for making this!

**Comment by Osspial** [comment]
@ElPumpo whoops, missed that bit. I've corrected that.

**Comment by ElPumpo** [comment]
You missed yet again 👀 
image

**Pull request closed without merging**